### PR TITLE
feat: add carusel

### PR DESCRIPTION
**Pull request by KseniaKuntsevich**
change photo gallery

### CODE REVIEW

File: src/components/Carusel/index.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+
+import './index.css';
+
+function Carusel({ list, description }) {
```
Comment (Xelsoid) [inline src/components/Carusel/index.js]: typo mistake 
"carousel"

File: src/components/ImageGallery/index.js [hunk 2]
```diff
@@ -1,24 +1,25 @@
 import React from 'react';
-import ImageGallery from 'react-image-gallery';
+import Carusel from '../Carusel/index';
```
Comment (Xelsoid) [inline src/components/ImageGallery/index.js]: the same

File: src/components/Carusel/index.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
+import React from 'react';
```
Comment (Xelsoid) [inline src/components/Carusel/index.js]: Please add Carousel component on the SB side as well

**Review by Xelsoid — CHANGES_REQUESTED** [review]

**Review by Xelsoid — APPROVED** [review]

File: src/components/Carusel/index.js (hunk 3 above)
Comment (KseniaKuntsevich) [reply src/components/Carusel/index.js]: Done

**Pull request merged**